web/package.json

```diff
@@ -1,23 +1,29 @@
 {
-  "name": "openkf",
-  "version": "0.0.0",
-  "private": true,
-  "scripts": {
-    "dev": "vite",
-    "build": "vite build",
-    "preview": "vite preview"
-  },
-  "dependencies": {
-    "@element-plus/icons-vue": "^2.1.0",
-    "axios": "^1.4.0",
-    "echarts": "^5.4.2",
-    "element-plus": "^2.3.7",
```

Comment:
suggest to use acro-design or Tdesign
element-plus is up to date

Reply:
Tdesign looks better.